src/mocks/assetsList.ts

```diff
@@ -1,14 +1,18 @@
-import { Asset } from '@cognite/sdk';
+import { Asset } from '@cognite/sdk-alpha/dist/src/types/types';
 import { AdvancedSearch } from '../interfaces';
 
 export const vmateDba = 'wmate_dba.wmt_location';
 export const randomTime = '1999-09-01 07:00:00';
+export const randomTimeDateObj = new Date('2002-02-02 22:22:22');
 export const valDescription = 'Valhall plattform';
 export const gasCompressionString = 'GAS COMPRESSION AND RE-INJECTION (PH)';
 export const wmtag = 'wmate_dba.wmt_tag';
 
 export const SKA: Asset = {
   id: 8129784932439587,
+  lastUpdatedTime: randomTimeDateObj,
```

Comment:
For some reason those 3 keys (`lastUpdatedTime`, `createdTime` and `depth`) marked as duplicated in travis test log.

Reply:
fixed